chat.py

```diff
@@ -15,6 +15,8 @@
 PRINT_ONLY = CONFIG['chat']['print_only'].get(bool)
 COST_ALERT_THRESHOLD = CONFIG['chat']['cost_alert_threshold'].get(float)
 COST_ALERT_EMOJI = CONFIG['chat']['cost_alert_emoji'].get()
+COST_BELOW_VALUE_PROJECTS_EXCLUDED = CONFIG['chat']['minimun_cost_warning'].get()
```

Comment:
- Use the same name for the constant and the config name.
- get should return a float
- There's a typo in `minimun`
- Maybe a good name would be: `COST_MIN_TO_NOTIFY` instead of `COST_BELOW_VALUE_PROJECTS_EXCLUDED`

Reply:
these changes are made in commit ff21bcdc9826c136fb88af83b9cac9b9b53ddd03f